backend/src/api/model/series.rs

```diff
@@ -45,16 +58,22 @@ impl_from_db!(
     select: {
         series.{
             id, opencast_id, state,
-            title, description, created,
-            metadata, read_roles, write_roles,
+            title, description,
+            metadata, created,
+            read_roles, write_roles,
         },
+        updated: "case \
+            when ${table:series}.updated = '-infinity' then null \
+            else ${table:series}.updated \
+        end",
```

Comment:
There is nothing wrong with this solution, but just to show you an alternative: you can say `row.updated::<postgres_types::Timestamp::<DateTime<Utc>>>()` and then match on that. But it probably doesn't even save code...

Reply:
Interesting. I tried something along those lines but couldn't get it working, though I image it shouldn't be that hard. But if you don't mind the current version, I'll just leave it at that.